lib/Ed25519VerificationKey2020.js

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
 import * as base58btc from 'base58-universal';
-import * as ed25519 from '@stablelib/ed25519';
+import key from './ed25519';
```

Comment:
```suggestion
import ed25519 from './ed25519.js';
```

Reply:
renaming from `key` to `ed25519` would have broken the CI so did the commit with all the changes here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/202ce82d3b3839738563ededf6a6da865918c50a